src/main/scala/github4s/Implicits.scala

```diff
@@ -0,0 +1,63 @@
+package github4s
+
+import cats.std.FutureInstances
+import cats.std.future._
+import cats.{ Monad, Id, Eval, MonadError }
+import github4s.free.interpreters.Interpreters
+import scala.concurrent.{ ExecutionContext, Future }
+
+object Implicits {
+
+  implicit val evalMonadError: MonadError[Eval, Throwable] = new MonadError[Eval, Throwable] {
+
+    override def pure[A](x: A): Eval[A] = Eval.now(x)
+
+    override def map[A, B](fa: Eval[A])(f: A ⇒ B): Eval[B] = fa.map(f)
+
+    override def flatMap[A, B](fa: Eval[A])(ff: A ⇒ Eval[B]): Eval[B] =
+      fa.flatMap(ff)
+
+    override def raiseError[A](e: Throwable): Eval[A] =
+      Eval.later({ throw e })
+
+    override def handleErrorWith[A](fa: Eval[A])(f: Throwable ⇒ Eval[A]): Eval[A] =
+      Eval.later({
+        try {
+          fa.value
+        } catch {
+          case e: Throwable ⇒ f(e).value
+        }
+      })
+  }
+
+  implicit def idMonadError(implicit I: Monad[Id]): MonadError[Id, Throwable] = new MonadError[Id, Throwable] {
```

Comment:
Note that the instance of `MonadError[Id, Throwable]` doesn't fullfill the laws of MonadError since exceptions cannot be captured. Do we want to provide it as part of the library knowing this?